src/components/policy/src/cache_manager.cc

```diff
@@ -313,6 +313,7 @@ bool CacheManager::AddDevice(const std::string& device_id,
   CACHE_MANAGER_CHECK(false);
   policy_table::DeviceParams& params =
       (*(pt_->policy_table.device_data))[device_id];
+  params.mark_initialized();
```

Comment:
@vlantonov Are you sure that params should be marked as `initialized` if add just `device_id` without adding its parameters?